### PR TITLE
DB-1737

### PR DESCRIPTION
**Pull request by sbtourist**

### CODE REVIEW

**Review by stef1927 — APPROVED** [review]
Looks great @sbtourist !

File: transport-native-epoll/src/main/java/io/netty/channel/epoll/AIOContext.java [hunk 1]
```diff
@@ -106,13 +110,15 @@ private int submitBatch(Batch<?> batch, boolean pendingLast) {
             Batch<?> toSubmit = batch.split(0, currentIndex);
             try {
                 Native.submitAIOReads(this, eventFd.intValue(), batch.fileDescriptor.intValue(), toSubmit.requests);
-            }  catch (Throwable e) {
+            } catch (Throwable e) {
```
Comment (stef1927) [inline transport-native-epoll/src/main/java/io/netty/channel/epoll/AIOContext.java]: If the submit fails `outstandingNr` should be decreased

**Review by stef1927 — COMMENTED** [review]
I've reviewed the latest changes, all seems correct except for one small thing.

**Comment by stef1927** [comment]
It needs rebasing before merging, otherwise +1. 

The version should already be `4.1.13.10` and the batch visibility change is already merged. So perhaps the version should be changed to `4.1.13.11`.

**Comment by sbtourist** [comment]
Rebased and updated version, now merging, thanks for the review!

**Pull request merged**